specification/sql/resource-manager/readme.md

```diff
@@ -108,7 +110,6 @@ input-file:
 - Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelOperations.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelPoolOperations.json
-- Microsoft.Sql/preview/2017-10-01-preview/capabilities.json
```

Comment:
What's the reason for removing this?

Reply:
So that location capabilities are now referenced to the updated swagger json in 2018-06-01-preview